src/index.js

```diff
@@ -707,6 +721,16 @@ class State extends EventEmitter {
 	}
 
 	/**
+	 * Send an arbitrary message to the named channel. The type of the message will be coerced to
+	 * a Max type in the Max application by mira.channel
+	 * @param {String} channel - The name of the mira.channel objects that should receive this message
+	 * @param {Number|String|Array<Number|String>|Object} message - the message to send
```

Comment:
should read
 @param {Number|String|Array<Number>|Array<String>|Object}

right?